todo/src/App.js

```diff
@@ -1,29 +1,50 @@
-// import logo from './logo.svg';
 import {useState} from 'react';
-
-import logo from './tek_logo.png';
 import './App.css';
-import Todo from './Components/Todo';
+import TodoList from './Components/TodoList';
+import TodoForm from './Components/Form'
+
+
 
 function App() {
 
-  const [todos, setTodos] = useState(["eat","sleep","code","pray","repeat"]);  //sample todo items.  these will need to be changed in your app.  Just filler todos, although they are quite important!
+    const [todos, setTodos] = useState([]); 
+  
+    const deleteHandler = (id) => {
+        setTodos(todos.filter(item => item.item !== id))
+    }
+  
+    const deleteAll = () => {
```

Comment:
If you have a function that is being used from only one component, that function should be there and not in the App. Like DeleteAll should be in TodoList. This breaks the single responsibility principle.